Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -679,4 +659,37 @@ private async Task SaveParticipantScore(Submission submission)
             .MapCollection<T>()
             .ToPagedResultAsync(DefaultSubmissionResultsPerPage, page);
     }
+
+    private async Task<PagedResult<SubmissionForPublicSubmissionsServiceModel>> GetPublicSubmissions(
+        SubmissionForPublicSubmissionsServiceModel model)
+    {
+        var user = this.userProviderService.GetCurrentUser();
+
+        if (user.IsAdminOrLecturer)
+        {
+            return await this.submissionsData.GetLatestSubmissions<SubmissionForPublicSubmissionsServiceModel>(
+                DefaultSubmissionsPerPage, model.PageNumber);
+        }
+
+        var modelResult = new PagedResult<SubmissionForPublicSubmissionsServiceModel>();
+
+        modelResult.Items = await this.submissionsData.GetLatestSubmissions<SubmissionForPublicSubmissionsServiceModel>(
+            DefaultSubmissionsPerPage);
+
+        return modelResult;
+    }
+
+    private async Task<PagedResult<SubmissionForPublicSubmissionsServiceModel>> GetPendingSubmissions(int page)
```

Comment:
These methods seem rendundant for now. Use the data methods in `GetSubmissions(string type, int page)` directly.